GenderPayGap.WebUI/Views/Viewing/EmployerDetails/Employer.cshtml

```diff
@@ -87,102 +85,157 @@
                             LinkText = "Who is required to report, and the deadlines",
                             SummaryPartial = "EmployerDetails/WhoNeedsToReport"
                         })
+                        <p>
+                            For example, if the snapshot date is 5 April 2017 but the deadline is 4 April 2018 then the reporting period is 2017 - 2018.
+                        </p>
 
                     </div>
                 </div>
-                @foreach (Return report in organisation.GetRecentReports(Global.ShowReportYearCount))
-                {
-                    <div class="grid-row">
-                        <div class="column-full">
-                            <table>
-                                <tbody>
-                                    <tr>
-                                        <td style="border-bottom: none; width: 300px;">
-                                            <h3 class="heading-small govuk-!-margin-top-0">@report.GetReportingPeriod() Reporting year</h3>
-                                        </td>
-                                    </tr>
-                                    @if (report.IsLateSubmission || report.IsVoluntarySubmission())
+                @await Html.GovUkTable(new TableGovUkViewModel
+                    {
+                        Head = new List<TableCellViewModel>
+                        {
+                            new TableCellViewModel
+                            {
+                                Text = " Reporting year", 

```

Comment:
I know that's how it's used elsewhere in this file but why does this need a space at the start?